linux-proposal.md

```diff
@@ -0,0 +1,252 @@
+corporate partners. Securing consistent funding via grants has proven
+challenging given the lumpy nature of grant allocations and the limited
+number of people who can devote their time to grant writing for Juptyer.
+These inconsistent levels of funding make it very difficult to maintain
+a stable backbone for the project that enables long-term vision and
+strategic planning and execution. Similarly, we have faced challenges in
+raising consistent funding from corporations.
+
+Additionally, Jupyter needs **improved operational support**. Jupyter
+needs a higher level of support for operating (legal/trademark,
+marketing, fundraising, accounting, contracts, internships, events, dev
+ops, etc.) than is currently available from NumFOCUS. The EC\'s
+responsibility is to serve Jupyter\'s pressing needs today and tomorrow;
+currently, the EC is spending a significant amount of its time on
+operational matters instead of fundraising and other activities that
+support Jupyter's long-term mission. At this time, and after extensive
+conversations with NumFOCUS and time spent by Ana Ruvalcaba taking deep
+dives into operational matters, the EC\'s assessment is that the best
+course of action is to secure operational support with a proven partner
+that can provide it today. *Our assessment is that the Linux Foundation
+is such an ideal partner for Jupyter today*.
+
+Finally, Jupyter needs to be able to hire and manage **full-time
+staff**, including roles such as an Executive Director, CI/devops,
+program managers, test engineering, documentation writing, and release
+engineering. These require a sustained and full-time effort, beyond what
+is reasonable for volunteers or can be funded via project-specific
+initiatives and/or grants. In the past, we have funded, hired, and
+managed these types of roles through universities (Cal Poly State
+University, San Luis Obispo and UC Berkeley), but that is not
+sustainable for the institution or the staff who seek a long term career
+path associated with Jupyter, rather than with specific corporate or
+academic organizations. Again, Jupyter needs sustainable funding and
+operational support to hire, retain, and manage contributors in these
+types of roles.
+
+# So, what would be changing?
+
+We have spent time and effort in discussions with the Linux Foundation
+team, to ensure that this transition introduces the smallest possible
+changes to Jupyter\'s operations while meeting our above goals.
+
+Under this proposal, **Jupyter keeps its current governance structure
+and transfers from NumFOCUS to LF Charities 501(c)(3). An additional
+funding vehicle will be created called the Jupyter Foundation under the
+Linux Foundation 501(c)(6) providing corporations a well known and
+proven method to support the Jupyter Project.**
+
+In order to manage this new resource (the Jupyter Foundation), we
+propose to create a new committee (JFC), populated by representatives of
```

Comment:
Does `JFC` stands for Jupyter Funding Committee?

Reply:
Yes, this is what we meant here. The name is something that may change as things progress.